web/components/proposal-table.tsx

```diff
@@ -91,12 +91,7 @@ const columns: ColumnDef<ProposalListParserOutcome[number]>[] = [
       </div>
     ),
     enableHiding: false,
-  },
-  {
-    accessorKey: 'category',
-    filterFn: (row, value) => {
-      return value.includes(row.original.category);
-    },
```

Comment:
This removes the ability to filter by proposal category.
Was this wanted? @robinbb

Reply:
No, this seems to be a typo that removes an entire brace-delimited stanza by accident. Please correct, @jstavh

Reply:
@52 Note that filtering by category does not currently work for me on https://mina.vote. What about you?

Reply:
@robinbb Weird - it does work for me. Which browser are you using?

Reply:
@52 made this change in the recent commit